tests/foreman/cli/test_import.py

```diff
@@ -59,6 +60,62 @@ def build_csv_file(rows=None):
     return file_name
 
 
+def modify_csv_values(csv_file, new_data=None):
+    """Build CSV with modified key values provided as an argument
+    in order to randomize the dataset with keeping the org_id mappings
+
+    :param csv_file: A string. The path to a CSV file that resides
+    on a remote server.
+
+    :param new_data: A list of dictionary objects. Each object has to
+    contain an id corresponding to organization_id of the record being
+    modified and the key-value pairs of attributes to be changed for
+    the particular records
+
+    :returns: A string. The path to a locally created CSV file
```

Comment:
Can you provide a brief example of what form the `new_data` argument should take?

    {???}